art_show/templates/art_show_applications/art_show_form.html

```diff
@@ -139,7 +139,7 @@
 </div>
 
 <div class="form-group">
-  <label for="website" class="col-sm-3 control-label optional-field">Website URL</label>
+  <label for="website" class="col-sm-3 control-label">Website URL</label>
```

Comment:
This change already exists, my local branches are just completely hosed for some godawful reason.